src/specs/NativeVideoDecoderProperties.ts

```diff
@@ -0,0 +1,17 @@
+import type {TurboModule} from 'react-native';
+import {TurboModuleRegistry} from 'react-native';
+import type {Int32} from 'react-native/Libraries/Types/CodegenTypes';
+
+export interface NativeVideoDecoderPropertiesSpec extends TurboModule {
+  getWidevineLevel: () => Promise<Int32>;
+  isCodecSupported: (
+    mimeType: string,
+    width: Int32,
+    height: Int32,
+  ) => Promise<'unsupported' | 'hardware' | 'software'>;
+  isHEVCSupported: () => Promise<'unsupported' | 'hardware' | 'software'>;
+}
```

Comment:
For some funny reason interfaces that extends `TurboModule` must have be named `Spec` (otherwise codegen is throwing error)
```suggestion
interface Spec extends TurboModule {
  getWidevineLevel: () => Promise<Int32>;
  isCodecSupported: (
    mimeType: string,
    width: Int32,
    height: Int32,
  ) => Promise<'unsupported' | 'hardware' | 'software'>;
  isHEVCSupported: () => Promise<'unsupported' | 'hardware' | 'software'>;
}

export interface NativeVideoDecoderPropertiesSpec extends Spec {}
```